### DISCUSSION TITLE
Is there no way to disable the new discography design now?

### DISCUSSION THREAD

**Discussion by ffsmyusernameistaken** [SpotX-Official/SpotX#469] [Question]
If so, in which version was this added as a permanent feature? I want new lyrics but I don't want this discography.

**Comment by amd64fox** [comment]
do you mean the discography on the artist page?

**Reply by ffsmyusernameistaken** [reply]
Ой, я забыла, что можно по-русски писать. Да, вид дискографии. Раньше это была экспериментальная фича, а теперь постоянная?

**Reply by amd64fox** [reply]
эта exp'а осталась, но теперь она меняет только подачу в самой странице артиста, а структуру дискографии уже не поменять в новых версиях:

https://github.com/amd64fox/SpotX/assets/62529699/ced0b21f-0167-4101-bf3a-33d897fbc6cd
